---
fixes:
  - |
    Fixed an issue with the :func:`~retworkx.generators.heavy_hex_graph`,
    :func:`~retworkx.generators.directed_heavy_hex_graph`,
    :func:`~retworkx.generators.heavy_square_graph`,
    and :func:`~retworkx.generators.directed_heavy_square_graph` generator
    functions. When the input parameter ``d`` was set to 1 these functions
    would previously raise a ``pyo3_runtime.PanicException`` instead of
    returning the expected graph (a single node). 
    Fixed `#452 <https://github.com/Qiskit/retworkx/issues/452>`__
